frontend/src/Receiving/ReceivingListModal.tsx

```diff
@@ -1,99 +1,163 @@
 import React, { useState, useContext, FC } from 'react';
-import { Modal, Header, Button } from 'semantic-ui-react';
-import { ReceivingContext } from '../context/ReceivingContext';
+import { Modal, Button, Form, List, Header } from 'semantic-ui-react';
+import { ReceivingContext, Trade } from '../context/ReceivingContext';
 import Price from '../common/Price';
+import { useFormik } from 'formik';
+import sum from '../utils/sum';
 
-interface Props {
-    cashTotal: number;
-    creditTotal: number;
+interface Props {}
+
+interface FormValues {
+    customerName: string;
+    customerContact: string;
 }
 
-const ReceivingListModal: FC<Props> = ({ cashTotal, creditTotal }) => {
+const initialFormValues: FormValues = {
+    customerName: '',
+    customerContact: '',
+};
+
+// TODO: Extract and generalize this
```

Comment:
Relevant. Going to tackle this when we move to refactor all formik forms with `useFormik`